src/pages/saksbehandling/sakintro/Utbetalinger.tsx

```diff
@@ -1,93 +1,182 @@
     // Tidligste utbetaling må være etter eller lik den første neste måned (nåværende backend impl).
     const kanStanses =
-        sak.utbetalingerKanStansesEllerGjenopptas === KanStansesEllerGjenopptas.STANS &&
-        sak.utbetalinger.some(
+        kanStansesEllerGjenopptas === KanStansesEllerGjenopptas.STANS &&
+        utbetalingsperioder.some(
             (u) =>
                 !DateFns.isBefore(DateFns.parseISO(u.tilOgMed), DateFns.startOfMonth(DateFns.addMonths(new Date(), 1)))
         );
 
     return (
-        <div className={styles.container}>
-            <ul className={styles.utbetalinger}>
-                {sak.utbetalinger.map((u) => (
-                    <li key={u.id}>
-                        <Panel border>
-                            <div>
-                                <p>Fra og med: {u.fraOgMed}</p>
-                                <p>Til og med: {u.tilOgMed}</p>
-                                <p>Beløp: {u.beløp}</p>
-                                <p>Type: {u.type}</p>
-                            </div>
-                        </Panel>
-                    </li>
-                ))}
-            </ul>
-            {kanStanses && (
-                <Knapp
-                    onClick={() => {
-                        if (!RemoteData.isPending(stansUtbetalingerStatus)) {
-                            dispatch(
-                                sakSlice.stansUtbetalinger({
-                                    sakId: props.sak.id,
-                                })
-                            );
-                        }
-                    }}
-                    spinner={RemoteData.isPending(stansUtbetalingerStatus)}
-                    className={styles.stansUtbetalinger}
-                >
-                    Stans utbetalinger
-                </Knapp>
-            )}
-            {sak.utbetalingerKanStansesEllerGjenopptas === KanStansesEllerGjenopptas.GJENOPPTA && (
-                <Knapp
-                    onClick={() => {
-                        if (!RemoteData.isPending(gjenopptaUtbetalingerStatus)) {
-                            dispatch(
-                                sakSlice.gjenopptaUtbetalinger({
-                                    sakId: props.sak.id,
-                                })
-                            );
-                        }
-                    }}
-                    spinner={RemoteData.isPending(gjenopptaUtbetalingerStatus)}
-                    className={styles.stansUtbetalinger}
+        <div className={styles.utbetalingContainer}>
+            <Undertittel className={styles.tittel}>
+                {intl.formatMessage({ id: 'display.stønadsperioder.tittel' })}
+            </Undertittel>
+            <Panel border>
+                <div className={styles.stønadsperiodeHeader}>
+                    <Undertittel>
+                        <FormattedDate value={utbetalingsperioder[0].fraOgMed} month="2-digit" year="numeric" /> -{' '}
```

Comment:
Nitpick : Ser i filen att det är en blandning av bruk av `<FormattedDate .. />` och `intl.formatXXX()`. Kanske digg att bara bruka en av de?

Reply:
(Jag lutar mest mot `intl.formatXXX` i template strings, ser inte helt fördelen med att ha de som komponenter`

Reply:
Opprinnelig fant jeg ikke noe på docs som viste hvordan dato formatering ble gjort for intl.formatDate, så brukte komponents måten. Men brukte litt tid på å søke opp, og har endret det til å være konsistent med de andre tilfellene.